config/config.properties

```diff
@@ -72,6 +72,15 @@ pravegaservice.zkURL=localhost:2181
 # Valid values: Positive integer.
 #pravegaservice.zkSessionTimeoutMs=10000
 
+# Enable security for the ZK client.
+#pravegaservice.secureZK=false
+
+# Location of trust store for secure connection to ZK.
+#pravegaservice.zkTrustStore=
```

Comment:
Just to check, is in the plan to execute system in secure mode? If it has not been done yet, would it be interesting to perform at least one system test build of this PR with security enabled?